packages/bridge/src/vl/animation-controls/AnimationControls.ts

```diff
@@ -1,7 +1,8 @@
-import { VL_BINARY_EXPRESSION_TYPES, VLAnimation, VLViz } from '../../types';
+import { VL_BINARY_EXPRESSION_TYPES, VLAnimation, VLTimeZoneDate, VLViz } from '../../types';
 import { select } from '../../util/Utils';
 
 export class AnimationControls {
+  protected _formatCb: (value) => void;
```

Comment:
Being specific, the value should be `Date | number` and should return a `string`, right?

And why is this protected? Is this class being inherited somewhere?